io.github.noxworld_dev.OpenNox.yml

```diff
@@ -0,0 +1,446 @@
+    version: *runtime-version
+    no-autodownload: true
+
+  org.freedesktop.Platform.GL32:
+    directory: lib/i386-linux-gnu/GL
+    version: *gl-version
+    versions: *gl-versions
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d;vulkan/implicit_layer.d
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+
+finish-args:
+  - --allow=multiarch
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to talk to the network:
+  - --share=network
+  # OpenGL acceleration
+  - --device=dri
+  # Sound access
+  - --socket=pulseaudio
+  # Needs to load Nox data from somewhere:
+  - --filesystem=home # TODO: list all dirs that we usually scan?
+
+modules:
+  - name: bundle-setup
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/i386-linux-gnu
+      - mkdir -p /app/lib/debug/lib/i386-linux-gnu
+      - mkdir -p /app/lib/i386-linux-gnu/GL
+      - install -Dm644 ld.so.conf /app/etc/ld.so.conf
+    sources:
+      - type: file
+        dest-filename: ld.so.conf
+        url: data:/app/lib32%0A/app/lib/i386-linux-gnu%0A
+  - name: opennox
+    buildsystem: simple
+    build-options:
+#      build-args:
+#        - --share=network
+      prepend-pkg-config-path: /app/lib32/pkgconfig:/usr/lib/i386-linux-gnu/pkgconfig
+      ldflags: -L/app/lib32
+      prepend-path: /usr/lib/sdk/golang/bin:/usr/lib/sdk/toolchain-i386/bin
+    build-commands:
+      - install -Dm 644 io.github.noxworld_dev.OpenNox.metainfo.xml /app/share/metainfo/io.github.noxworld_dev.OpenNox.metainfo.xml
+      - install -Dm 644 ./res/app/opennox_256.png /app/share/icons/hicolor/256x256/apps/io.github.noxworld_dev.OpenNox.png
+      - install -Dm 644 io.github.noxworld_dev.OpenNox.desktop /app/share/applications/io.github.noxworld_dev.OpenNox.desktop
+      - install -Dm 644 io.github.noxworld_dev.OpenNox.HD.desktop /app/share/applications/io.github.noxworld_dev.OpenNox.HD.desktop
+      - cd src && GOARCH=386 CC=i686-unknown-linux-gnu-gcc go run ./internal/noxbuild client client-hd
+      - install -Dm 755 ./src/opennox /app/bin/opennox
+      - install -Dm 755 ./src/opennox-hd /app/bin/opennox-hd
+    sources:
+
+      # Main engine source
+      - type: git
+        url: https://github.com/noxworld-dev/opennox
+        # tag: v1.8.12 # TODO: release
+        commit: d3b74deca498000222f2816ea1c195c68e86d92d
+
+      # Resources
+      - type: git
+        url: https://github.com/noxworld-dev/opennox-content
+        commit: c262619501e6c76b208adbf66d604de07ddbddaa
+        dest: ./res
+
+      # Flatpak files
+      - type: file
+        path: io.github.noxworld_dev.OpenNox.desktop
+      - type: file
+        path: io.github.noxworld_dev.OpenNox.HD.desktop
+      - type: file
+        path: io.github.noxworld_dev.OpenNox.metainfo.xml
```

Comment:
These files should be ideally maintained in your repo. They are not specific to Flatpak.